All-in-one setup/createform.ps1

```diff
@@ -627,20 +628,32 @@ try {
 
         $exchangeSession = Connect-ExchangeOnline -Credential $credential -ShowBanner:$false -ShowProgress:$false -TrackPerformance:$false -ErrorAction Stop 
 
-        Hid-Write-Status -Event Information -Message "Successfully connected to Office 365"
+        Write-Information -Message "Successfully connected to Office 365"
+
+        $Log = @{
+            Action            = "MailboxPermissions" # optional. ENUM (undefined = default) 
```

Comment:
Action cannot be MailboxPermissions. See https://docs.helloid.com/en/service-automation/delegated-forms/delegated-form-powershell-scripts/audit-logging-in-delegated-form-powershell-scripts.html for a list of actions

Reply:
Changed to "UpdateAccount"